### ISSUE TITLE
prevent lookahead bias

### ISSUE THREAD

**Issue by zohaad** [alexanderpei/DeepEarnings#1]
In `trn_NetCompustat.py` you need to pass in `shuffle=False` to `sklearn.model_selection.train_test_split`

**Comment by alexanderpei** [comment]
Hey, thank you for the suggestion. Actually, I need to do more than just removing shuffling to look into this since each company is loaded one after the other into the data set. Will let you know.

**Comment by zohaad** [comment]
You also need to add a `date < filingDate` restriction to the split
